assets/javascripts/discourse/controllers/event-rsvp.js.es6

```diff
@@ -14,23 +15,23 @@ export default Ember.Controller.extend({
     const type = this.get('type');
     const topic = this.get('model.topic');
 
-    let usernames = topic.get(`event_${type}`);
+    let user_ids = topic.get(`event_${type}`);
 
-    if (!usernames || !usernames.length) return;
+    if (!user_ids || !user_ids.length) return;
 
     let userList = [];
 
-    usernames.forEach((username, index) => {
-      User.findByUsername(username).then((user) => {
-        userList.push(user);
-
-        if (userList.length == usernames.length) {
-          this.setProperties({
-            userList,
-            loadingList: false
-          })
-        }
-      })
+    ajax('/calendar-events/rsvp/get_users', {
```

Comment:
@fzngagan The Discourse convention for route formatting is hyphenated, i.e. get-users. Also, the convention is to not include actions like "get". So the path here should be:
```
/calendar-events/rsvp/users
```